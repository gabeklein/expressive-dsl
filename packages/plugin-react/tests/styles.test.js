const test = require("./_adapter");

test("basic style", `
  const Component = () => do {
    color: green;

    <this>Hello World</this>
  };
`);