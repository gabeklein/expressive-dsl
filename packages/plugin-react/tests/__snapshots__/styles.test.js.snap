// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic style 1`] = `
"const {
  default: Styles
} = require(\\"@expressive/react\\");

const {
  createElement: create
} = require(\\"react\\");

const Component = () => create(\\"div\\", {
  \\"className\\": \\"Component_3cs\\"
}, \\"Hello World\\");

Styles.include(\`
	.Component_3cs { 
		color: green;
	}
\`);"
`;

exports[`basic style 2`] = `
"import Styles from \\"@expressive/react\\";
import React from \\"react\\";

const Component = () => <div className=\\"Component_3cs\\">Hello World</div>;

Styles.include(\`
	.Component_3cs { 
		color: green;
	}
\`);"
`;
